ui/components/app/beta-header/beta-header.test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { fireEvent } from '@testing-library/react';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import mockState from '../../../../test/data/mock-state.json';
+import { renderWithProvider } from '../../../../test/lib/render-helpers';
+import BetaHeader from '.';
+
+const mockHideBetaHeader = jest.fn();
+
+jest.mock('../../../store/actions', () => {
+  return {
+    hideBetaHeader: () => {
+      mockHideBetaHeader();
+    },
+  };
+});
+
+describe('Beta Header', () => {
+  afterEach(() => {
+    mockHideBetaHeader.mockClear();
+  });
+
+  const store = configureMockStore([thunk])(mockState);
```

Comment:
This is now being shared between tests, which could result in one test affecting another. Better to construct this per-test or in `beforeEach` to ensure the tests are isolated.

Reply:
Good shout.